docs/specs/tripgo.swagger.yaml

```diff
@@ -2147,9 +2163,13 @@ definitions:
               type: string
             price:
               type: number
+            duration:
+              type: number
+              description: in minutes
           required:
             - label
```

Comment:
Might be good to say that even though label is required, it's often empty. Which makes me wonder why make it required and not just leave it out.

Reply:
Yes. I am going to update this so the only required one is `price`.